bower.json

```diff
@@ -29,7 +29,7 @@
   "ignore": [],
   "dependencies": {
     "polymer": "Polymer/polymer#^1.2.0",
-    "iron-autogrow-textarea": "PolymerElements/iron-autogrow-textarea#^1.0.0",
+    "iron-autogrow-textarea": "PolymerElements/iron-autogrow-textarea#^1.0.11",
```

Comment:
This change isn't needed -- ^1.0.0 uses the latest version, and your change doesn't break if you have a lower version than `1.0.11`. Since you're just styling the placeholder, it would be a no-op if you had a lower version, so it's still fine